docs/en/setup/configuration/process_discovery/vm.md

```diff
@@ -0,0 +1,68 @@
+# VM Process Detector
+
+The VM process detector could be detected any process in the VM with finders.
+Each finder could define a regex to filter which processes need to be monitored, and the metadata of the process when the report to the backend.
+
+## Configuration
+
+| Name | Default | Environment Key | Description |
+|------|---------|-----------------|-------------|
+| process_discovery.vm.active | false | ROVER_PROCESS_DISCOVERY_VM_ACTIVE | Is active the VM mode to detect processes. |
+| process_discovery.vm.period | 3s | ROVER_PROCESS_DISCOVERY_VM_PERIOD | The period to detect the process. |
+| process_discovery.vm.finders | | | It could be define multiple finders to find the process. It only provide one finder by default. |
+| process_discovery.vm.finders.match_cmd_regex | | ROVER_PROCESS_DISCOVERY_VM_FINDER_MATCH_CMD_REGEX | Use regex string to locate the process from the command line of the process. |
+| process_discovery.vm.finders.layer | OS_LINUX | ROVER_PROCESS_DISCOVERY_VM_FINDER_LAYER | The Layer need to relate to the process entity |
```

Comment:
Replace all `need to` to `define`/`declare` or something.

Reply:
Updated.